source/index.txt

```diff
@@ -1,80 +1,247 @@
+            including:
+
+            - :method:`db.collection.updateOne()`
+            - :method:`db.collection.updateMany()`
+            - :method:`db.collection.replaceOne()`
+
+            For more information and examples, refer to :ref:`mongosh-update`.
+
+         .. step:: Delete Documents
+
+            :binary:`~bin.mongosh` supports common delete opererations, 
+            including:
+
+            - :method:`db.collection.deleteMany()`
+            - :method:`db.collection.deleteOne()`
+
+            For more information and examples, refer to :ref:`mongosh-delete`.
+
+         .. step:: Run Aggregation Pipelines
+
+            You can run aggregation pipelines in :binary:`~bin.mongosh` 
+            using the :method:`db.collection.aggregate()` method. For more 
+            information and examples, refer to :ref:`mdb-shell-aggregation`.
+
+      .. image:: /images/mongosh-plp/Technical_ACTION_SearchDocument_Thumbnail_BS_Mist.png
+         :alt: Abstract illustration showing CRUD and aggregation operations
+
+   .. tab:: Administer and Manage
+      :tabid: administer
+
+      .. procedure::
+
+         .. step:: Manage Databases and Collections
+
+            View information about databases, create collections or views,
+            or drop databases - all from your shell. See all 
+            :ref:`Database Methods <mongosh-database-methods>`.
+
+            Perform collection operations, create or delete indexes, or 
+            explain queries with :ref:`Collection Methods 
+            <mongosh-collection-methods>`.
+
+         .. step:: Administer Servers
+
+            Manage :ref:`replication <mongosh-replication-methods>` or 
+            :ref:`sharding <mongosh-sharding-methods>` from the comfort 
+            of your shell.
+
+            Check server status with a variety of :ref:`Server Status Methods 
+            <mongosh-server-status-methods>`.
+
+         .. step:: Manage Users and Roles
+
+            Create or update roles, define and update privileges, or drop 
+            roles using :ref:`Role Management Methods 
+            <mongosh-role-management-methods>`.
+
+            Create and update users, authenticate users, and manage user
+            roles with :ref:`User Management Methods 
+            <mongosh-user-management-methods>`. 
+
+      .. image:: /images/mongosh-plp/Technical_SOFTWARE_Terminal(3)_Spot.png
+         :alt: Abstract illustration showing a terminal performing operations
 
-.. default-domain:: mongodb
+   .. tab:: Write Scripts
+      :tabid: scripts
 
-.. contents:: On this page
-   :local:
-   :backlinks: none
-   :depth: 1
-   :class: singlecol
+      .. procedure::
 
-The |mdb-shell|, ``mongosh``, is a fully functional JavaScript and 
-Node.js 16.x |repl| environment for interacting with MongoDB 
-deployments. You can use the |mdb-shell| to test queries and operations 
-directly with your database.
+         .. step:: Perform Common Functions with Scripts
```

Comment:
Can we consider breaking this section into the following parts?

1) Running DB scripts against mongosh for CRUD or administrative tasks e.g if you have a JS file that seeds synthetic / mock data into your DB in your dev env or staging, you can run the js file with `mongosh <jsfile>.js` 

2) Create custom helper functions with .mongoshrc (see [this](https://preview-mongodbdacharyc.gatsbyjs.io/mongodb-shell/DOCSP-32028/write-scripts/#execute-code-from-a-configuration-file)). The use case we should describe here could be something like: "if you convert date string to ISO format often for queries, you can create a function in `.mongoshrc` `function toISO(dateString) { return new Date(dateString).toISOString(); }` and call e.g `toISO("06/07/2017") in mongosh."

3) Use or publish snippets. You can [pull existing snippets](https://preview-mongodbdacharyc.gatsbyjs.io/mongodb-shell/DOCSP-32028/snippets/working-with-snippets/#install-snippet-packages). You can also create & publish new snippets for your custom use case, example: a snippet that validates data that you are importing daily as a cron, you can publish this as a snippet so your dev team can have access to it. You can publish either to the community registry or configure a private registry.

4) Code scoping and other considerations

Reply:
I've rewritten the content on this tab similar to your suggestions here, @GaurabAryal - is this what you had in mind?